academic-services/includes/ma-subscription-receive-data.md

```diff
@@ -0,0 +1,96 @@
+* Permission to write and add role assignment to the storage account. These permissions exist in the Owner role.
+
+## Gather the information that you need
+
+Before you begin, you should have these items of information:
+
+   :heavy_check_mark:  The name of your Azure Storage (AS) account from [Get Microsoft Academic Graph on Azure storage](./get-started-setup-provisioning#note-azure-storage-account-name).
+
+   :heavy_check_mark:  The name of the container in your Azure Storage account from [Get Microsoft Academic Graph on Azure storage](./get-started-setup-provisioning#create-a-blob-container).
+
+## Sign in to the Azure portal
+
+Sign in to the [Azure portal](https://portal.azure.com/).
+
+## Open your invitation
+
+1. You can open the invitation from your email client or directly from the Azure portal. 
+
+   * To open invitation from email, check your inbox for an invitation from your data provider. The invitation is from Microsoft Azure, titled **Azure Data Share invitation from <yourdataprovider@domain.com>**. Click on **View invitation** to see your invitation in Azure. 
+
+   * To open invitation from Azure portal directly, search for **Data Share Invitations** in Azure portal. This action takes you to the list of Data Share invitations.
+
+1. Select the share you would like to view. 
+
+   ![List of Invitations](../media/receive-data/invitations.png "List of invitations") 
+
+## Accept the invitation
+
+1. Make sure all fields are reviewed, including the **Terms of Use**. If you agree to the terms of use, you'll be required to check the box to indicate you agree. 
+
+   ![Terms of use](../media/receive-data/terms-of-use.png "Terms of use") 
+
+1. Under *Target Data Share Account*, select the Subscription and Resource Group that you'll be deploying your Data Share into. 
+
+   For the **Data Share Account** field, select **Create new** if you don't have an existing Data Share account. Otherwise, select an existing Data Share account that you'd like to accept your data share into. 
+
+   For the **Received Share Name** field, you may leave the default specified by the data provide, or specify a new name for the received share. 
+
+   Once you've agreed to the terms of use and specified a Data Share account to manage your received share, Select **Accept and configure**. A share subscription will be created. 
+
+   This action takes you to the received share in your Data Share account. 
+
+   If you don't want to accept the invitation, Select **Reject**. 
+
+   ![Accept options](../media/receive-data/accept-options.png "Accept options") 
+
+## Configure received share
+
+Follow the steps below to configure where you want to receive data.
+
+1. Select the **Datasets** tab. Check the box next to the dataset you'd like to assign a destination to. Select **+ Map to target** to choose a target data store. Check both `mag` and `makes` to receive MAG and MAKES data. Or check one dataset to receive only MAG data or only MAKES data.
```

Comment:
If this is not shared documentation, should we focus on the MAKES selection here?